Gitweb:     http://git.kernel.org/linus/4c49b12853fbb5eff4849b7b6a1e895776f027a1
Commit:     4c49b12853fbb5eff4849b7b6a1e895776f027a1
Parent:     67178767b936fb47a3a5e88097cff41ccbda7acb
Author:     Arjan van de Ven <arjan@infradead.org>
AuthorDate: Fri Nov 13 21:47:33 2009 -0800
Committer:  Ingo Molnar <mingo@elte.hu>
CommitDate: Sat Nov 14 10:23:52 2009 +0100

    perf_event: Fix invalid type in ioctl definition
    
    u64 is invalid in userspace headers, including ioctl
    definitions; use __u64 instead
    
    Signed-off-by: Arjan van de Ven <arjan@linux.intel.com>
    Cc: <stable@kernel.org>
    LKML-Reference: <20091113214733.7cd76be9@infradead.org>
    Signed-off-by: Ingo Molnar <mingo@elte.hu>
---
 include/linux/perf_event.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

--- a/include/linux/perf_event.h
+++ b/include/linux/perf_event.h
@@ -219,7 +219,7 @@ struct perf_event_attr {
 #define PERF_EVENT_IOC_DISABLE		_IO ('$', 1)
 #define PERF_EVENT_IOC_REFRESH		_IO ('$', 2)
 #define PERF_EVENT_IOC_RESET		_IO ('$', 3)
-#define PERF_EVENT_IOC_PERIOD		_IOW('$', 4, u64)
+#define PERF_EVENT_IOC_PERIOD		_IOW('$', 4, __u64)
 #define PERF_EVENT_IOC_SET_OUTPUT	_IO ('$', 5)
 
 enum perf_event_ioc_flags {
